src/bundle/Resources/public/ts/shared/BaseBadge.ts

```diff
@@ -0,0 +1,31 @@
+import Base from './Base';
```

Comment:
If you resigned from base and inheritance in react part, you can drop BaseBadge from twig as well, and put this file directly as Badge.ts
If we'll need to add inheritance it will still be easy to revert this change. :)